Model/Catalog/Layer/Url/Strategy/PathSlugStrategy.php

```diff
@@ -551,7 +551,18 @@ public function getCategoryFilterSelectUrl(
          This is needed because the categoryUrlPath contains the store code in some cases and the directUrl as well.
          These two are the only unique parts in this situation and so need to be removed.
          */
-        return implode('/', array_unique(explode('/', $url)));
+
+        $explode = explode('/', $url);
+        $lastpart = array_last($explode);
```

Comment:
@ah-net How is this supposed to work and even tested? `array_last` method does not exist in php. 
After installing latest release application is broken:

`Call to undefined function Tweakwise\Magento2Tweakwise\Model\Catalog\Layer\Url\Strategy\array_last()`

Reply:
@bramstroker 

It seems that the array last function comes from an package instead of php (which is assumed). Forgot too check where the function comes from when adding this with autocomplete. The package (weew/helpers-array) was automatically installed on all our test env's, so we didn't catch this.

I will fix this, and release an new version

Reply:
@ah-net Ok, but please don't require a seperate package just for this one liner which can be easily solved using native PHP. You can just use `end` or various different approaches to do this.

Reply:
Fixed in version 5.3.3. Used end instead of array_last